pkg/services/apiserver/aggregator/aggregator.go

```diff
@@ -137,6 +137,7 @@ func CreateAggregatorConfig(commandOptions *options.Options, sharedConfig generi
 		},
 	}
 
+	klog.Infof("Legacy Client Cert Auth is: %s", commandOptions.KubeAggregatorOptions.LegacyClientCertAuth)
```

Comment:
Locally, I get this log, but in the cloud, Loki isn't seeing it with `log_level` set to 10 under the `[grafana-apiserver]` section.